### ISSUE TITLE
Mouse positions in replay are inaccurate

### ISSUE THREAD

**Issue by stefandesu** [Pernosco/DOMRec#5]
For me (using Brave), the mouse positions in the replay are offset. Looking at the CSS, it seems like the offset is 15 pixels (no matter how small or big I make the cursor). Maybe you can fix it in CSS, but for me, it works well if I adjust the following piece of code:

https://github.com/Pernosco/DOMRec/blob/b296fdac887bf8c1808c03a63a6833d7881fa0b4/domrec.js#L651-L657

```js
// TODO: Dynamically determine offset (or fix in CSS).
const offset = 15
this.cursor.style.left = (x + offset) + "px";
this.cursor.style.top = (y + offset) + "px";
```

**Comment by Junkern** [comment]
Yeah, I have noticed the offset, aswell. Also, the height itself is set correctly, but still some of the bottom pixels are cut off (at least in Firefox).

Also, the mouse offset looks totally different in Firefox, compared to Brave/Chrome.

**Comment by stefandesu** [comment]
With my fix above, it seems to work for me in both Brave and Firefox. In Safari, the mouse position is also correct, but there seems to be another issue:

```
TypeError: undefined is not an object (evaluating 'rangeRect.left')
```
